src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -209,6 +209,12 @@ extension ExposureSubmissionCoordinator {
 			}
 		)
 
+		#if DEBUG
+		if isUITesting {
```

Comment:
Why not leave this in in release?

Reply:
The [initial reference solution](https://stackoverflow.com/questions/38117410/how-can-i-set-accessibilityidentifier-to-uialertcontroller/40688141#40688141) used a private api (`__representer`) but not this specific line.

Also, I've just discovered that we can simply do `acceptAction.accessibilityIdentifier = "fooo"` :)

Reply:
I'll try to set acceptAction.accessibilityIdentifier